libs/designer-ui/src/lib/editor/base/plugins/tokenpickerbutton/index.tsx

```diff
@@ -31,6 +31,7 @@ export const TokenPickerButton = ({ showOnLeft, openTokenPicker }: TokenPickerBu
   const [editor] = useLexicalComposerContext();
   const [anchorKey, setAnchorKey] = useState<NodeKey | null>(null);
   const boxRef = useRef<HTMLDivElement>(null);
+  const panel = document.getElementsByClassName('ms-Panel-scrollableContent')[0];
```

Comment:
Just to catch if it can't find the element for whatever reason
```suggestion
  const panel = document.getElementsByClassName('ms-Panel-scrollableContent')?.[0];
```

Reply:
This should be fine, it should always return the first element or undefined - and not generate any error. Just tested locally, 
![image](https://user-images.githubusercontent.com/84426548/235221382-74b70ce0-f604-4f0d-9134-c27faf4551d2.png)